### DISCUSSION TITLE
Unexpected token '<' | My project works via localhost but not when I push it to a sever

### DISCUSSION THREAD

**Discussion by Alex-C-Codes** [jeffreylanters/react-unity-webgl#519] [Help Wanted!] [answered]
Hey,

My React.js and Unity based website works when viewed via the localhost. However, when I try to start the site via a server, the game doesn't load and I see this error:

Unexpected token '<'

Attached is a screenshot showing the pathways I'm using to get the game to show on the homepage.

Any help on why this might not be working on a server but works locally would be greatly appreciated.

<img width="953" alt="Screen Shot 2023-07-24 at 11 45 25 AM" src="https://github.com/jeffreylanters/react-unity-webgl/assets/54649908/f4fb34fb-9156-4a45-8180-e07d21561607">

Thanks,
Alex-C

**Comment by jeffreylanters** [comment, marked as answer]
This is probably caused by the space in the directory name. Try replacing it with a dash.

**Reply by Alex-C-Codes** [reply]
Hey, thanks for the quick response. I removed the space and made it one word, and I'm still having issues with the server-side deploy. Here's a link and screenshot if that's helpful:

Link to react js game: 
https://towngame-6e61aa6ed47b.herokuapp.com

Screenshot showing updated directory name:
<img width="1096" alt="Screen Shot 2023-07-24 at 1 52 32 PM" src="https://github.com/jeffreylanters/react-unity-webgl/assets/54649908/4383e222-4ab5-4340-a0b2-b2b46d2625ab">

**Reply by jeffreylanters** [reply]
Your casing in incorrect. Make sure the casing matches your actual filenames.

**Reply by Alex-C-Codes** [reply]
Hey Jeffrey, this works now - thank you so much!!!
